samples/azure-quantum/adapting-code/adapting-qiskit-sample.ipynb

```diff
@@ -0,0 +1,160 @@
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# 1. Download a sample from Qiskit.org\r\n",
+        "Let's say you want to try the QAOA sample: https://qiskit.org/textbook/ch-applications/qaoa.html\r\n",
+        "\r\n",
+        "1. Locate the **Download as Jupyter Notebook** hyperlink on the right side of the page\r\n",
+        "2. Download the Jupyter Notebook\r\n",
+        "3. Open your Azure Quantum workspace > **Notebooks**\r\n",
+        "4. Click **My notebooks** > ...\r\n",
+        "5. Click on the **Upload new** button and select the notebook your just downloaded\r\n",
+        "6. Open the newly imported notebook"
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# 2. Locate the aer_simulator\r\n",
+        "Most samples are configured to run by default against the aer_simulator, which is a great way to get started.\r\n",
+        "\r\n",
+        "1. Locate the cell that has a line like the following:\r\n",
+        "**backend = Aer.get_backend('aer_simulator')**\r\n",
+        "2. Comment out that line\r\n",
+        "\r\n"
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# 3. Replace the aer_simulator with an Azure Quantum backend\r\n",
+        "\r\n",
+        "1. Add a new code cell prior to the cell that had backend = Aer.get_backend('aer_simulator')\r\n",
+        "2. Add the following code\r\n",
+        "\r\n",
+        "```\r\n",
+        "from azure.quantum.qiskit import AzureQuantumProvider\r\n",
+        "provider = AzureQuantumProvider (\r\n",
+        "    resource_id = \"\",\r\n",
+        "    location = \"\"\r\n",
+        ")\r\n",
+        "\r\n",
+        "# Create IonQ simulator and QPU backends\r\n",
+        "ionq_simulator_backend = provider.get_backend(\"ionq.simulator\")\r\n",
+        "ionq_qpu_backend = provider.get_backend(\"ionq.qpu.aria-1\")\r\n",
+        "\r\n",
+        "# Create Rigetti simulator and QPU backends\r\n",
+        "rigetti_simulator_backend = provider.get_backend(\"rigetti.sim.qvm\")\r\n",
+        "rigetti_qpu_backend = provider.get_backend(\"rigetti.qpu.aspen-m-3\")\r\n",
+        "\r\n",
+        "# Create Quantinuum simulator and QPU backends\r\n",
+        "quantinuum_simulator_backend = provider.get_backend(\"quantinuum.sim.h1-2sc\")\r\n",
+        "quantinuum_qpu_backend = provider.get_backend(\"quantinuum.qpu.h1-2\")\r\n",
+        "\r\n",
+        "backend = ionq_simulator_backend\r\n",
+        "```\r\n",
+        "\r\n"
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
```

Comment:
Should we add instructions on how to install additional packages if needed using !pip install syntax? Some of the qiskit libs for ML use pytorch. That may need to be installed based on what they are trying to do. Pytorch itself can be 1GB by itself, and more if you pull in optional packages, so it's not great to add to the base image, unless we have a lot of usage of pytorch.

Reply:
Thanks for the suggestions. Made the changes.